packages/frr/build-frr.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+CWD=$(pwd)
+set -e
+
+FRR_SRC=frr
+
+if [ ! -d ${FRR_SRC} ]; then
+    echo "FRR source directory does not exists, please 'git clone'"
+    exit 1
+fi
+
+# VyOS requires some small FRR Patches - apply them here
+# It's easier to habe them here and make use of the upstream
+# repository instead of maintaining a full Fork.
+# Saving time/resources is essential :-)
+cd ${FRR_SRC}
+
+PATCH_DIR=${CWD}/patches
+
+for patch in $(ls ${PATCH_DIR})
+do
+    echo "I: Apply FRR patch: ${PATCH_DIR}/${patch}"
+    patch -p1 < ${PATCH_DIR}/${patch}
+    git add *
+    git -c user.name='Vyos Custom Build' -c user.email='ci@vyos.io' commit -m "Applied patch: ${patch}"
```

Comment:
This line will become problematic if a user tries to build the package and has his `.gitconfig` mounted into the build container.

I happen to be such a crazy user - and I do not want to commit as user: "Vyos Custom Build" in the future.

Also it should be VyOS instead of Vyos.

Reply:
Except this small change it looks good to me!

Reply:
I was wrong infact - this only applies to this very one commit but not the entire config - despite the commands look the same.